### PR TITLE
feat: create cli command to create documentai batch jobs

### PR DESCRIPTION
**Pull request by steveoh**
🎉

### CODE REVIEW

File: row_run.py [hunk 1]
```diff
@@ -44,10 +45,27 @@ def mosaic_all_circles():
         row.format_time(perf_counter() - job_start),
     )
 
+
+def ocr_all_mosaics():
+    """the main function to execute when cloud run starts the ocr job"""
     job_start = perf_counter()
 
+    inputs = SimpleNamespace(
```
Comment (eneemann) [inline row_run.py:53]: praise: I haven't really done it before, but I like the idea of using SimpleNamespace here.

File: row_cli.py [hunk 2]
```diff
@@ -135,8 +126,35 @@ def main():
...
+            task_index=int(args["--task-index"]),
+            task_count=int(args["--instances"]),
+            total_size=int(args["--file-count"]),
+            project_number=int(args["--project"]),
+            processor_id=int(args["--processor"]),
+        )
+
+        results = row.ocr_all_mosaics(inputs)
+
+        print(f"operation finished with status {results}")
```
Comment (eneemann) [inline row_cli.py:155]: question: `results` is the dataframe passed out of `ocr_all_mosaics`, right? What will that look like when printed here?  Is it just for the CLI and not the logging?

**Review by eneemann — COMMENTED** [review]
I think this all looks good to me.  I had a question about the print statement with the results, but it's more for my understanding.  Thanks for putting this together.

File: row_run.py (hunk 1 above)
Comment (steveoh) [reply row_run.py:53]: yeah, just too many individual variables.

File: row_cli.py (hunk 2 above)
Comment (steveoh) [reply row_cli.py:155]: it's actually a list of lists...

File: row.py [hunk 3]
```diff
@@ -122,6 +127,94 @@ def process_all(job_name, input_bucket, output_location, file_index, task_index,
...
+            file_index (str): the path to the folder containing an `index.txt` file listing all the images in a bucket.
+                            `gs://bucket-name`
+            task_index (int): the index of the task running
+            task_count (int): the number of containers running the job
+            total_size (int): the total number of files to process
+            project_number (int): the number of the gcp project
+            processor_id (str): the id of the documentai processor
+
+    Returns:
+        A pandas dataframe with the results of the OCR
```
Comment (steveoh) [inline row.py:147]:
```suggestion
        A list of lists with the results of the OCR
```

File: row_cli.py (hunk 2 above)
Comment (eneemann) [reply row_cli.py:155]: Oh, that's right.  So it just prints the list (of lists) to the console when running the CLI?

**Review by eneemann — APPROVED** [review]
Let's give it a shot!

**Pull request merged**